lib/addons/p5.dom.js

```diff
@@ -58,12 +58,23 @@
    * </code></div>
    *
    */
-  p5.prototype.select = function (e) {
+  p5.prototype.select = function (e, p) {
     var res;
     var str;
+    var container = document;
+    if (typeof p === 'string'){
+      if(p[0] !== '#'){
+        container = document.getElementById(p);
+      }else {
+        p = p.slice(1);
+        container = document.getElementById(p);
+      }
```

Comment:
also wondering, if they input a string without a # before, should we assume it's an id, or maybe it makes more sense to assume it's a tag name and just pick the first element returned by getElementsByTagName()?
I think this aligns more closely with the way the first argument to this function works.